docs/sources/features/panels/stat.md

```diff
@@ -0,0 +1,78 @@
++++
+draft = "false"
+date = "2020-02-19"
+title = "Stat panel"
+description = "Stat panel docs"
+keywords = ["grafana", "enter", "keywords", "here"]
+type = "docs"
++++
+
+# Stat panel
+
+{{< docs-imagebox img="/img/docs/v66/stat_panel_dark3.png" max-width="1025px" caption="Stat panel" >}}
+
+The stat panel is designed to show a big single stat values with an optional graph sparkline. You can control
+background or value color using thresholds.
+
+## Display options
+
+* Show
+  * `Calculation` - Calculate a show a single value like min or max based on all rows
```

Comment:
As suggested by @oddlittlebird in another review - use dots at the end of sentence